cmd/sink/expr/cel.go

```diff
@@ -0,0 +1,312 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package expr
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/google/cel-go/cel"
+	"github.com/google/cel-go/checker/decls"
+	celext "github.com/google/cel-go/ext"
+	"github.com/kubearchive/kubearchive/cmd/operator/api/v1alpha1"
+	"github.com/kubearchive/kubearchive/pkg/files"
+	"gopkg.in/yaml.v3"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const globalKey = "_global"
+
+var ErrNoGlobal = errors.New("no global expressions exist")
+
+type Filters struct {
+	*sync.RWMutex
+	archive map[string]cel.Program
+	delete  map[string]cel.Program
+	path    string
+}
+
+// Returns a Filters struct with an empty archive and delete slice
```

Comment:
IINM, conventionally the go doc comments good practices suggest to use the name of the exported function in the beginning of the doc:
```suggestion
// EmptyFilters returns a Filters struct with an empty archive and delete slice
```
See examples in the [Go Doc Comments Guide](https://go.dev/doc/comment#func)

Reply:
done